tests/integration/targets/utils_ipaddr_filter/tasks/ip4_hex.yaml

```
@@ -14,3 +14,13 @@
 - name: Assert result for ip4_hex.
   ansible.builtin.assert:
     that: "{{ result1 == 'c0:a8:01:05' }}"
+
+- name: Ip4_hex validate input
+  ansible.builtin.set_fact:
+    result1: "{{ '555.444.333.999'|ansible.utils.ip4_hex }}"
```

```suggestion
    result1: "{{ '555.444.333.999' | ansible.utils.ip4_hex }}"
```
Adding proper Jinja spacing so that ansible-lint doesn't warn us about it.